Sources/Penny/Handlers/InteractionHandler.swift

```diff
@@ -486,17 +486,26 @@ private extension InteractionHandler {
         }
     }
     
-    func handleLinkCommand(options: [InteractionOption]) throws -> String {
+    func handleGitHubCommand(options: [InteractionOption]) throws -> (any Response)? {
+        let discordId = try (event.member?.user).requireValue().id
         let first = try options.first.requireValue()
-        let subCommand = try LinkSubCommand(rawValue: first.name).requireValue()
-        let id = try (first.options?.first).requireValue().requireString()
-        switch subCommand {
-        case .discord:
-            return "This command is still a WIP. Linking Discord with Discord ID '\(id)'"
-        case .github:
-            return "This command is still a WIP. Linking Discord with Github ID '\(id)'"
-        case .slack:
-            return "This command is still a WIP. Linking Discord with Slack ID '\(id)'"
+        let subcommand = try GitHubSubCommand(rawValue: first.name).requireValue()
+        let githubUsername = first.options?.first?.value
+
+        if subcommand == .link, githubUsername == nil {
+            return "You need to provide a GitHub username to link your account."
+        }
+
+        switch subcommand {
+        case GitHubSubCommand.link:
+            let clientID = try Constants.ghOAuthClientId.requireValue()
+            let url = "https://github.com/login/oauth/authorize"
+            let githubLink = "\(url)?client_id=\(clientID)"
+            return "Use this to link your GitHub account to your Penny account: \(githubLink)"
```

Comment:
So we could be fancier. First of all, you need to add this extension to the end of the file (lines 1000-1200) where `protocol Response` and its extensions are:
```swift

extension Payloads.EditWebhookMessage: Response {
    func makeResponse(isEphemeral: Bool) -> Payloads.InteractionResponse {
        Logger(label: "Payloads.EditWebhookMessage.makeResponse").error(
            "This method is unimplemented and must not be called"
        )
        return .channelMessageWithSource(
            .init(content: "Oops, something went wrong")
        )
    }

    func makeEditPayload() -> Payloads.EditWebhookMessage {
        self
    }

    var isEditable: Bool { true }
}
```
Then you can return a payload with a button:
```swift
return Payloads.EditWebhookMessage(
                embeds: [.init(
                    description: "Click the link below to authorize Vapor",
                    color: .vaporPurple
                )],
                components: [[
                    .button(.init(label: "Authorize", url: githubLink))
                ]]
            )
```